test/unit/data_hygiene/statistics_announcement_unpublisher_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+require 'gds_api/test_helpers/router'
+
+module DataHygiene
+  class StatisticsAnnouncementUnpublisherTest < ActiveSupport::TestCase
+    include GdsApi::TestHelpers::Router
+
+    setup do
+      @fake_logger = mock()
+      @fake_logger.stubs(:info)
```

Comment:
There's also `NullLogger.instance` that you could use instead of a mock. Might be more robust.